ee/supported-platforms.md

```diff
@@ -65,16 +91,10 @@ and third-party ecosystem solution briefs.
 
 ## Docker Enterprise Edition release cycles
 
-Docker EE is released quarterly. Releases use a time-based versioning
-scheme, so for example, Docker EE version 17.03 was released
-in March 2017. For schedule details, see
-[Time-based release schedule](/engine/installation/#time-based-release-schedule).
-
-Each Docker EE release is supported and maintained for one year and
+Each Docker EE release is supported and maintained for 24 months, and
```

Comment:
@mataneja is this really 24 months?

Reply:
Yes, going forward our support lifecycle has been extended to 24 months